hooks/post_gen_project.py

```diff
@@ -1,9 +1,211 @@
+            "enable_strict_checks":
+                "Require branches to be up to"
+                + " date before merging (y/n)? [n]: ",
+            "enforce_admins":
+                "Require admins to oblige by status checks (y/n)? [n]: ",
+            "enforce_code_owner_review":
+                "Require a designated code owner to"
+                + " approve pull requests (y/n)? [n]: ",
+            "enable_issues":
+                "Enable the GitHub Issues features"
+                + " on the repository (y/n)? [n]: ",
+            "enable_merge_commits": "Allow merge commits (y/n)? [n]: ",
+            "enable_squash_merging": "Allow squash merging (y/n)? [n]: ",
+            "enable_rebase_merging": "Allow rebase merging (y/n)? [n]: ",
+        }
+
+        # Multiple entry questions
+        DICT_QUESTIONS = {
+            "topics": {
+                "start_question": "Add a topic to the repository (y/n)? [n]: ",
+                "init_loop": "What topic would you like to add? "
+            },
+            "required_status_checks": {
+                "start_question":
+                    "Add required checks for merging into"
+                    + " the branch (y/n)? [n]: ",
+                "init_loop":
+                    "What check would you like to add? "
+            },
+            "restricted_pr_teams": {
+                "start_question":
+                    "Restrict which teams can dismiss push requests"
+                    + " (y/n)? [n]: ",
+                "init_loop":
+                    "What team would you like to restrict? "
+            },
+            "restricted_push_teams": {
+                "start_question":
+                    "Restrict who can push to the branch (y/n)? [n]: ",
+                "init_loop":
+                    "What team would you like to restrict? "
+            }
+        }
+
+        directory = "terraform/"
+        os.mkdir(directory)
+        token_q = (
+            "Enter the personal access token for "
+            + "creating the organizational repo: ")
+        os.environ['GITHUB_ORGANIZATION'] = (
+            "{{ cookiecutter.github_username }}")
+        os.environ['GITHUB_TOKEN'] = input(token_q).lower()
+
+        terraform_vars = {
+            "name": "{{ cookiecutter.module_name }}",
+            "description": "{{ cookiecutter.short_description }}"
+        }
+
+        for var, question in YN_QUESTIONS.items():
+            choice = check_choice(question)
+            terraform_vars[var] = str(choice).lower()
+
+        for var, question in DICT_QUESTIONS.items():
+            choice = check_choice(question['start_question'])
+            if choice:
+                selection = multiple_choices(question['init_loop'])
+                terraform_vars[var] = json.dumps(selection)
+
+        collaborators = get_collaborators()
+        if collaborators:
+            terraform_vars['access_teams'] = (
+                json.dumps(collaborators).replace(":", "="))
```

Comment:
What's the magic replace doing here? A comment would help on that

Reply:
Also, the `json.dumps` seems odd here. We should be able to just collect all the values in python objects, then output to json _only_ when writing the file...

Reply:
ACK. this looks like a holdover from when the .tfvars file was being written with hcl. Didn't catch it b/c I was happy pathing my tests